### DISCUSSION TITLE
Popover

### DISCUSSION THREAD

**Discussion by RAIbrahim360** [quasarframework/quasar#17062] [Ideas / Proposals] [answered]
Add popover like https://www.npmjs.com/package/floating-vue. I don't find any similar there

**Comment by rstoenescu** [comment, marked as answer]
QMenu

**Reply by RAIbrahim360** [reply]
It doesn't have option to not close when mouse over the popover. And it doesn't have click trigger

**Reply by rstoenescu** [reply]
Use a v-model on it and check all the props (see the API card in docs).
